### ISSUE TITLE
"The model does not have any trainable weights." in Bijector of MAF code

### ISSUE THREAD

**Issue by dguhanus** [whitead/dmol-book#252]
Dear Sir,
I am interested in normalizing flow priors. I was running the tensorflow code. Initially I was suffering from not being able to pass Keras Layer as a function argument to Tensorflow function. Which I Had overcome using a wrapper. After that I am facing 
warnings.warn("The model does not have any trainable weights.")``

Here is the error that I have reproduced for your reference: https://github.com/dguhanus/Generative_Flow_Tensorflow/blob/main/Error_in_MAFlows.ipynb
